packaging/alvr_build_linux.sh

```diff
@@ -181,6 +181,7 @@ prep_fedora_server() {
     basePackages=(
         'dnf-utils'
         'git'
+        'cmake'
```

Comment:
Is this a new requirement for nvenc?

Reply:
No. Just deps for fedora. My distr. To build the server

Reply:
This should probably be added to the `BuildRequires` in `packaging/rpm/alvr.spec`; the packages in this script are _only_ added because they're needed for `yum-builddeps`.

Reply:
I see, ok Ill move